src/WebJobs.Script.WebHost/WebScriptHostBuilderExtension.cs

```diff
@@ -40,12 +41,15 @@ public static class WebScriptHostBuilderExtension
                 {
                     loggingBuilder.Services.AddSingleton<ILoggerFactory, ScriptLoggerFactory>();
 
-                    loggingBuilder.Services.AddSingleton<ILoggerProvider, SystemLoggerProvider>();
+                    loggingBuilder.AddWebJobsSystem();
 
                     ConfigureRegisteredBuilders(loggingBuilder, rootServiceProvider);
                 })
                 .ConfigureServices(services =>
                 {
+                    // We want to remove any registered IScriptEventManager so that the WebHost's will be used
+                    services.RemoveAll<IScriptEventManager>();
```

Comment:
@fabiocav -- wanted your take on this. The JobHost needs an IScriptEventManager, but I want it to live at the WebHost level when running in the WebHost. So here I remove any already-registered (and elsewhere I register it at the root provider). Is that an okay approach? I wasn't sure if we needed to create/dispose one of these per-JobHost. If so, I can always create some new interface that we register explicitly for WebHost-level events.

Reply:
Ignore this -- completely changed the approach so this isn't used anymore.